onnx/utils.py

```diff
@@ -10,6 +10,7 @@
 import onnx.helper
 import onnx.shape_inference
 from onnx import FunctionProto, ModelProto, NodeProto, TensorProto, ValueInfoProto
+from onnx.checker import MAXIMUM_PROTOBUF
```

Comment:
Import modules only

Reply:
I would avoid importing a class or a function from a module.

Import only the module instead to keep the name space clean and provide readers with more context on where its members come from. For example, readers don’t need to backtrack to see `sleep` is a function from `time`, as opposed to a function defined in the file. https://google.github.io/styleguide/pyguide.html#22-imports

Reply:
Thank you @justinchuby , I changed it to `onnx.checker.MAXIMUM_PROTOBUF `